<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openWidget</class>
 <widget class="QDialog" name="openWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#openWidget {
	background-image: url(:/new/prefix1/chess/99bOOOPIC52.jpg);
}</string>
  </property>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: black;
	background-color: #c0c1bb;	
	border: none;
	border-radius: 10px;

}

QPushButton:hover {
	color: black;
	background-color: #b99b7f;
	border: none;
	border-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string>自我对战</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>17</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: black;
	background-color: #c0c1bb;	
	border: none;
	border-radius: 10px;

}

QPushButton:hover {
	color: black;
	background-color: #b99b7f;
	border: none;
	border-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string>人机对战</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>17</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: black;
	background-color: #c0c1bb;	
	border: none;
	border-radius: 10px;

}

QPushButton:hover {
	color: black;
	background-color: #b99b7f;
	border: none;
	border-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string>连网对战</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
